works-cards.js

```diff
@@ -0,0 +1,32 @@
+import { workcards } from './data.js';
+
+const generateworkCardHTML = (workCard) => {
+  const { title, subtitle, tags } = workCard;
+  const tagsHTML = tags.map((tag) => `<li>${tag}</li>`).join('');
+  const cardHTML = `
+    <div class="works-card">
+      <div class="works-card-content">
+        <h3 class="works-card-title">${title}<span><br>${subtitle}</span></h3>
+        <div class="works-tags">
+          <ul>
+            ${tagsHTML}
+          </ul>
+        </div>
+        <button class="project-btn" onclick="openDetailView()">See Project</button>
+      </div>
+    </div>
+  `;
```

Comment:
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 


- Please blur the sub background when the popup is open, it will make the popup more like Figma 👍🏿
- It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 
- Make sure the max width of your popup windows is 920px as shown on the Figma design.
- You need to respect the text description font size, line-height, font family, .... and space padding or margin ex: between the 'Tech stack` (HTML, CSS, js, ...). `text description` and  the  `See live and See source` buttons

Your Implementation | Figma Design
:---:|:---:
![image](https://github.com/RileyManda/portfolio-website/assets/79058364/e67da3d4-7753-46fc-a3f9-d87a49095557) | ![181](https://user-images.githubusercontent.com/79058364/189106086-ad46721f-211a-41e9-97b7-421fb32231c2.png) 


- [ ]  Please do the same for the desktop size 👨🏻‍🎨 🎨 

- Actually, I don't see the full height of your page popups when I'm on the desktop size, please make sure 100% height is shown 👨🏻‍🎨

Your Implementation | Figma Design
:---:|:---:
![image](https://github.com/RileyManda/portfolio-website/assets/79058364/fab4328e-d02b-47e4-9983-2e00a70c07a3) | ![18](https://user-images.githubusercontent.com/79058364/189106145-f947cebe-3a5a-45a8-9851-ada6c6544c46.png)